modules/hooks/settings.js

```diff
@@ -7,8 +7,8 @@ export default function() {
         // Then, make sure that change is reflected in the counter on the combat tracker (if the update was made by a different user)
         if (setting.key == "wfrp4e.groupAdvantageValues")
         {
-            ui.notifications.notify(game.i18n.format("GroupAdvantageUpdated", {players : setting.value.players, enemies : setting.value.enemies}))
-
+            let message = game.i18n.format("GroupAdvantageUpdated", {players : setting.value.players, enemies : setting.value.enemies});
+            ui.notifications.notify(message)
```

Comment:
Is `message` used somewhere else? Not sure what the intention here is

Reply:
Its really hard for me to explain, but sometimes i saw very strange exceptions being thrown in this method - thats why i wanted to split it into atomic steps.